.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        compiler: [GCC, CLANG, ICC, ICX]
+        compiler: [GCC, CLANG, ICX, ICPX]
```

Comment:
isnt ICX the new C and ICPX the new C++ compiler? i.e. those are effectively the same compilers?